src/components/Select/Select.stories.tsx

```diff
@@ -13,6 +13,13 @@ export default {
     badges: ['1.2'],
     layout: 'centered',
   },
+  argTypes: {
+    multiple: {
+      table: {
+        disable: null,
+      },
+    },
+  },
```

Comment:
Was this breaking the storybook from rendering? Or just disabling because that arg doesn't work well with storybook UI?

Reply:
Hiding this one b/c we will have a forthcoming component for handling multiple selection and didn't want to have this lingering prop visible. More work to do, but closing a documentation mismatch for the moment.